eng/common/scripts/create-tags-and-git-release.ps1

```diff
@@ -6,7 +6,6 @@ param (
   # used by VerifyPackages
   $artifactLocation, # the root of the artifact folder. DevOps $(System.ArtifactsDirectory)
   $workingDirectory, # directory that package artifacts will be extracted into for examination (if necessary)
-  [ValidateSet("Nuget","NPM","PyPI","Maven")]
   $packageRepository, # used to indicate destination against which we will check the existing version.
```

Comment:
@scbedd @chidozieononiwu I wonder if we shouldn't change this to be the language instead of package registry?